libs/employees/src/lib/employees/employees.component.ts

```diff
@@ -13,7 +13,9 @@ export class EmployeesComponent implements OnInit {
 
   constructor(
     private employeesService: EmployeesService
-  ) {}
+  ) {
+    this.employeesService.getEmployees().subscribe()
```

Comment:
This should go inside of ngOnInit to confirm to Angular lifecycle

Reply:
Fixed